tests/manage/storageclass/test_create_multiple_sc_with_same_pool_name.py

```diff
@@ -0,0 +1,129 @@
+        *. Run IO on each app pod
+        """
+        # Create 5 Storage Classes with same pool name
+        sc_obj_list = []
+        sc_list = []
+        for i in range(1, 3):
+            log.info(f"Creating a {interface} storage class")
+            if interface == "RBD":
+                interface_type = constants.CEPHBLOCKPOOL
+                secret = self.rbd_secret_obj.name
+                interface_name = self.cbp_obj.name
+            else:
+                interface_type = constants.CEPHFILESYSTEM
+                secret = self.cephfs_secret_obj.name
+                interface_name = helpers.get_cephfs_data_pool_name()
+            sc_obj = helpers.create_storage_class(
+                interface_type=interface_type,
+                interface_name=interface_name,
+                secret_name=secret,
+            )
+            log.info(
+                f"{interface}StorageClass: {sc_obj.name} created successfully"
+            )
+            sc_obj_list.append(sc_obj)
+            sc_list.append(sc_obj.name)
+
+        # Create PVCs using each SC
+        pvc_obj_list = []
+        pvc_list = []
+        for i in range(1, 3):
+            log.info(f"Creating a PVC using {sc_list[i-1]}")
+            pvc_obj = helpers.create_pvc(sc_list[i - 1])
+            log.info(
+                f"PVC: {pvc_obj.name} created successfully using"
+                f" {sc_list[i-1]}"
+            )
+            pvc_list.append(pvc_obj.name)
+            pvc_obj_list.append(pvc_obj)
+
+        # Create app pod and mount each PVC
+        pod_obj_list = []
+        pod_list = []
+        for i in range(1, 3):
+            if interface == "RBD":
+                interface_type = constants.CEPHBLOCKPOOL
+            else:
+                interface_type = constants.CEPHFILESYSTEM
+
+            log.info(f"Creating an app pod and mount {pvc_list[i-1]}")
+            pod_obj = helpers.create_pod(
+                interface_type=interface_type, pvc_name=pvc_list[i - 1],
+                desired_status=constants.STATUS_RUNNING,
+                wait=True, namespace=defaults.ROOK_CLUSTER_NAMESPACE
+            )
+            log.info(
+                f"{pod_obj.name} created successfully and mounted "
+                f"{pvc_list[i-1]}"
+            )
+            pod_list.append(pod_obj.name)
+            pod_obj_list.append(pod_obj)
+
+        # Run IO on each app pod for sometime
+        for pod in pod_obj_list:
+            log.info(f"Running FIO on {pod.name}")
+            pod.run_io('fs', size='2G')
+
+        for pod in pod_obj_list:
+            fio_result = pod.get_fio_results()
+            log.info(f"IOPs after FIO for pod {pod.name}:")
+            log.info(
+                f"Read: {fio_result.get('jobs')[0].get('read').get('iops')}"
+            )
+            log.info(
+                f"Write: {fio_result.get('jobs')[0].get('write').get('iops')}"
+            )
+
+        # Cleanup
+        # Delete APP Pod's
```

Comment:
Please move the deletion of the resource to teardown.
For that, you can create a fixture that initializes and returns empty lists, one for each resource type. 
In its finalizer, it'll delete the resources created during the test.

As a reference, check out the resources fixture in 
https://github.com/red-hat-storage/ocs-ci/blob/98bd64e34817cc47f537bcc56e758bed979f3a2c/tests/manage/cluster/nodes/test_nodes_restart.py#L68

Reply:
Done

Reply:
Done